Research Papers/Shallow learning/README.md

```diff
@@ -304,6 +304,8 @@ This paper investigates a relatively new direction in Multiagent Reinforcement L
 
 <details> <summary> <a href="https://www.cs.cmu.edu/~sandholm/cs15-892F15/MarginalContributionEC05.pdf"> Marginal contribution nets: A compact representation scheme for coalitional games </a>by Ieong S, Shoham Y. In Proceedings of the 6th ACM Conference on Electronic Commerce, 2005. <a href="link">  </a> </summary> We present a new approach to representing coalitional games based on rules that describe the marginal contributions of the agents. This representation scheme captures characteristics of the interactions among the agents in a natural and concise manner. We also develop efficient algorithms for two of the most important solution concepts, the Shapley value and the core, under this representation. The Shapley value can be computed in time linear in the size of the input. The emptiness of the core can be determined in time exponential only in the treewidth of a graphical interpretation of our representation. <br> - </details>
 
+<details> <summary> <a href="https://cs.gmu.edu/~sean/papers/luke05tunable.pdf"> Tunably decentralized algorithms for cooperative target observation </a>Luke, Sean and Sullivan, Keith and Panait, Liviu and Balan, Gabriel. 4th International Joint Conference on Autonomous Agents and Multiagent Systems (AAMAS 2005), 2005. <a href="link">  </a> </summary> Multi-agent problem domains may require distributed algorithms for a variety of reasons: local sensors, limitations of communication, and availability of distributed computational resources. In the absence of these constraints, centralized algorithms are often more efficient, simply because they are able to take advantage of more information. We introduce a variant of the cooperative target observation domain which is free of such constraints. We propose two algorithms, inspired by K-means clustering and hill-climbing respectively, which are scalable in degree of decentralization. Neither algorithm consistently outperforms the other across over all problem domain settings. Surprisingly, we find that hill-climbing is sensitive to degree of decentralization, while K-means is not. We also experiment with a combination of the two algorithms which draws strength from each. <br> - </details>
```

Comment:
For the conference, you can just put "AAMAS, 2005". The same applies to all other papers. Just the "acronym, year" is good.

Reply:
I see :) I will update all the PRs to conform to this :+1: